### DISCUSSION TITLE
动态首页Safari标签页栏中显示的颜色

### DISCUSSION THREAD

**Discussion by 775720665** [the1812/Bilibili-Evolved#2434] [功能建议 / Ideas]
Safari浏览器更新到15之后，标签可以显示网页的主题颜色了。这个功能使用起来大部分时间还是可以增加体验感。
但是有的时候启动夜间模式的网页往往由于没有设置夜间模式的主题色，所以导致还是使用日间模式的主题色。
这就造成了很尴尬的局面。

<img width="2497" alt="WeChatf3f5bc0ebfc20db1a973a91c7edd9054" src="https://user-images.githubusercontent.com/48470968/138648035-28950d86-0e3e-469d-a89a-ba930cf0a241.png">

能否在动态首页修改主题色，让这个页面在夜间模式下也能有良好的视觉效果 ？

`<meta name="theme-color" content="#222222">`

<img width="2501" alt="WeChat13663adbc0d31ad11e30f38c0ea3e3d4" src="https://user-images.githubusercontent.com/48470968/138648060-08a32549-ed1e-4973-a029-f3b6bef453e1.png">

或者区分夜间和日间模式
`<meta name="theme-color" content="#4393E2" media="(prefers-color-scheme: light)">`
`<meta name="theme-color" content="#222222" media="(prefers-color-scheme: dark)">`

**Comment by Geeyun-JY3** [comment]
重复：#2384
